android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -876,6 +855,42 @@ private void initializePlayerCore(ReactExoplayerView self) {
         }
     }
 
+    private AdsMediaSource initializeAds(MediaSource videoSource, Source runningSource) {
+        AdsMediaSource mediaSourceWithAds = null;
```

Comment:
Initializing this variable to null is redundant because you immediately assign it a value later on.

Reply:
yes you are right, I also fix a potential issue with the source vs running source.
This is fixed !